### PR TITLE
Add `open System.Globalization` to be able to use `CultureInfo` class

### PR DESCRIPTION
**Pull request by onpikono**
I propose adding `open System.Globalization` to "hint" students and lead them to explore the `CultureInfo` class instead of manually formatting the date string. Of course, the latter remains as a possibility.

### CODE REVIEW

**Comment by ErikSchierboom** [comment]
The goal of this exercise is just to give a brief introduction to `DateTime` and completely gloss over timezones and cultures, as we have a later exercise that goes into exactly that.

The instructions contain the following:

> The tests will automatically set the culture to en-US - you don't have to set or specify the culture yourselves.

We should maybe rephrase this is a little bit, maybe mentioning that one _shouldn't_ specify the culture yourself. Additionally, it might be good to make this bit of text standout a bit more. We can use an `~~~~exercism/caution` code block for that: https://exercism.org/docs/building/markdown/markdown

Would you be interested in submitting a PR?

**Comment by onpikono** [comment]
> The goal of this exercise is just to give a brief introduction to `DateTime` and completely gloss over timezones and cultures, as we have a later exercise that goes into exactly that.
> 
> The instructions contain the following:
> 
> > The tests will automatically set the culture to en-US - you don't have to set or specify the culture yourselves.

Oh, I am sorry, I completely overlooked that one, I have not paid enough attention. That lead to my solution being unnecessarily complicated using `CultureInfo` while a simple string interpolation one-liner would suffice.

> 
> We should maybe rephrase this is a little bit, maybe mentioning that one _shouldn't_ specify the culture yourself. Additionally, it might be good to make this bit of text standout a bit more. We can use an `~~~~exercism/caution` code block for that: https://exercism.org/docs/building/markdown/markdown
> 
> Would you be interested in submitting a PR?

I did not read the instructions properly, otherwise I would not miss this part, it's right there. However, if you still feel like it would be an improvement, let me know and I will submit a PR.

**Comment by ErikSchierboom** [comment]
> I did not read the instructions properly, otherwise I would not miss this part, it's right there. However, if you still feel like it would be an improvement, let me know and I will submit a PR.

I do indeed think this is an improvement. You missing it clearly shows that it doesn't really stand out. A PR would be great!

**Comment by onpikono** [comment]
PR #992 added as discussed.

**Comment by ErikSchierboom** [comment]
image

Much better! Thanks.
